packages/back-end/src/routers/dashboards/dashboards.router.ts

```diff
@@ -0,0 +1,83 @@
+import express from "express";
+import { z } from "zod";
+import { wrapController } from "back-end/src/routers/wrapController";
+import { validateRequestMiddleware } from "back-end/src/routers/utils/validateRequestMiddleware";
+import {
+  createDashboardBlockInterface,
+  dashboardBlockInterface,
+} from "back-end/src/enterprise/validators/dashboard-block";
+import { dashboardEditLevel } from "back-end/src/enterprise/validators/dashboard-instance";
+import * as rawDashboardsController from "./dashboards.controller";
+
+const router = express.Router();
+const dashboardsController = wrapController(rawDashboardsController);
+
+const dashboardParams = z.object({ id: z.string() }).strict();
+export const createDashboardBody = z
+  .object({
+    experimentId: z.string(),
+    title: z.string(),
+    editLevel: dashboardEditLevel,
+    enableAutoUpdates: z.boolean(),
+    blocks: z.array(createDashboardBlockInterface),
+  })
+  .strict();
+
+export const updateDashboardBody = z
+  .object({
+    title: z.string().optional(),
+    editLevel: dashboardEditLevel.optional(),
+    enableAutoUpdates: z.boolean().optional(),
+    blocks: z
+      .array(z.union([createDashboardBlockInterface, dashboardBlockInterface]))
+      .optional(),
+  })
+  .strict();
+
+router.get(
+  "/",
+  validateRequestMiddleware({}),
```

Comment:
nit: Do we need this or can we skip it if we don't need to validate request params?